src/app/models.py

```diff
@@ -4,7 +4,7 @@
 # Create your models here.
 
 # Errand model
-class Errand(models.Model):
+class ErrandModel(models.Model):
```

Comment:
Each object of this class represents an Errand, adding Model to the name seems like an unnecessary change